### PR TITLE
feat: add project not initialized screen

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

The API has the notion of initialized/non-initialized projects. Given that when projects are created they can remain in an uninitialized state this behaviour needs to visually be reflected in the MC.

#### Description

CleanShot 2019-04-16 at 09 18 56@2x

### CODE REVIEW

File: packages/i18n/data/core.json [hunk 1]
```diff
@@ -47,6 +47,8 @@
   "ProjectNotFound.paragraph1": "The Project you are looking for either does not exists or you are not authorized to view it. Select another Project to view.",
   "ProjectNotFound.paragraph2": "Please contact your system administrator if you have any further questions.",
   "ProjectNotFound.title": "We could not find this Project",
+  "ProjectNotInitialized.paragraph1": "Initialization should not take longer than a few minutes. Please contact us at {mailto} in case the project does not get initialized.",
+  "ProjectNotInitialized.title": "Your project is not initialized yet",
```
Comment (montezume) [inline packages/i18n/data/core.json]:
```suggestion
  "ProjectNotInitialized.title": "Your project has not yet been initialized",
```

File: packages/application-shell/src/components/project-not-initialized/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const ProjectNotInitialized = React.lazy(() =>
+  import(
+    './project-not-initialized' /* webpackChunkName: "project-not-initialzied" */
```
Comment (montezume) [inline packages/application-shell/src/components/project-not-initialized/index.js]: nitpick: typo in webpack chunk name

~~ initialzied~~ initialized

File: packages/assets/images/project-not-initialized.svg [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (emmenko) [inline packages/assets/images/project-not-initialized.svg]: Is this the same as the "project-expired" svg? 🤔

Comment (tdeekens) [reply packages/assets/images/project-not-initialized.svg]: Yes, which works fine for now I think. It's the exact same hourglass we have in the async creation in the MC. So it speaks the same visuals.

**Comment by tdeekens** [comment]
⚠️ This needs an update in our gateway to fetch the field obviously ⚠️

File: packages/assets/images/project-not-initialized.svg (hunk 3 above)
Comment (montezume) [reply packages/assets/images/project-not-initialized.svg]: should we then rename the SVGs like we did the icons? 

~project-expeired~ hourglass?

Comment (emmenko) [reply packages/assets/images/project-not-initialized.svg]: Do we need to have 2 identical svgs then?

Comment (tdeekens) [reply packages/assets/images/project-not-initialized.svg]: I think we're speaking of the same thing again.

1. No, we do not need two SVGs
2. Yes, using the `ProjectSuspendedSVG` in the `ProjectNotInitialized` screeen feels odd
3. Yes, we can improve the naming.
4. Yes, Duplicating the SVG seems to be the least evil also as it is code split.

However, changing the naming is a breaking change to the assets package. We can also duplicate even more to not have it breaking and export generic and specific names. Howveer, I think that's a bigger evil again even.

Comment (emmenko) [reply packages/assets/images/project-not-initialized.svg]: Hmm ok, whatever

Comment (tdeekens) [reply packages/assets/images/project-not-initialized.svg]: Are you willing to go through a breaking change to have different namings in the assets package for the SVGs?

Comment (montezume) [reply packages/assets/images/project-not-initialized.svg]: Let's track is as a separate issue then, and do it in a followup as suggested by @tdeekens. We can ask design to come up with semantic names for all the assets?

Comment (tdeekens) [reply packages/assets/images/project-not-initialized.svg]: For future reference: https://github.com/commercetools/merchant-center-application-kit/issues/546

**Comment by tdeekens** [comment]
An updated mc-be is rolled out across all envs this should be able to safely hit master.

**Pull request merged**